CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ## [unreleased] - yyyy-mm-dd
 
+### Changed
+* NetworkTransform bandwidth optimizations applied to NetworkObject prefabs inside project (#836) Netcode for GameObjects v1.4.0 introduced bandwidth compression techniques to further reduce the bandwidth footprint of a NetworkTransform's message packet. Inside Boss Room, several NetworkTransforms have now their "Synchronize Transform" flag now disabled for their NetworkTransform, meaning that its NetworkTransform properties will not be synced when spawning and/or when late-joining clients connect. This is particularly useful if the NetworkObject is used more for management related tasks and has no spatial synchronization needs. Additionally, the base prefab for PCs and NPCs in-game, Character, had its NetworkTransform modified to now utilize half float precision, ie. "Use Half Float Precision" set to true. Its y position is also explicitly no longer synced. This results in a net 5 byte reduction in a NetworkTransform's sync packet. For more information, see [Netcode for GameObjects' v1.4.0 release notes](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/releases/tag/ngo%2F1.4.0).
```

Comment:
isn't that flag on NetworkObjects and not on NetworkTransform? if you have an NO with no spatial needs, why have a NT at all.
I also wouldn't use "packet" for NT, since UnityTransport will batch stuff in the background. It's not one network packet per state update.

Reply:
Oh good catch. That was meant to be `NetworkObjects have now their "Synchronize Transform" properties disabled for their NetworkObject component`. This should be mentioned as an "Additionally,..." comment. I'll fix.

I'll rephrase that as the NetworkTransform synchronization payload.

Reply:
still using "packet"?